### ISSUE TITLE
requests.exceptions.HTTPError: Error (400)

### ISSUE THREAD

**Issue by marioortizmanero** [felix-hilden/spotipy#35]
I was trying to use this API but I constantly get this error for some reason... The setup is quite minimal and it still happens:

```python
from spotipy import Spotify, Scope, scopes, Token, Credentials
from spotipy.util import prompt_for_user_token, RefreshingToken


client_id = '...'
client_secret = '...'
redirect_uri = 'http://localhost:8888/callback/'
scope = Scope(*scopes)

token = prompt_for_user_token(client_id, client_secret, redirect_uri, scope)

spt = Spotify(token)


print(spt.current_user_albums())
print(spt.playback_currently_playing())
```

Could it be because I'm passing to Spotify a RefreshingToken instead of a Token? I'm 100% sure that my keys are correct.

Log:
```
❯ py test.py
Opening browser for Spotify login...
Please paste redirect URL: http://localhost:8888/callback/?code=...
Traceback (most recent call last):
  File "test.py", line 14, in <module>
    print(spt.current_user_albums())
  File "/home/mario/.local/lib/python3.7/site-packages/spotipy/client/library.py", line 25, in current_user_albums
    return self._get('me/albums', market=market, limit=limit, offset=offset)
  File "/home/mario/.local/lib/python3.7/site-packages/spotipy/client/base.py", line 71, in _get
    return self._send(r)
  File "/home/mario/.local/lib/python3.7/site-packages/spotipy/client/base.py", line 61, in _send
    raise HTTPError(f'Error ({r.status_code}) in {r.url}', request=r)
requests.exceptions.HTTPError: Error (400) in https://api.spotify.com/v1/me/albums?market=from_token&limit=20&offset=0
```

Edit: turns out I'm dumb. I should have used `Spotify(self._token.access_token)` instead of `Spotify(self._token)`

**Comment by felix-hilden** [comment]
Yeah, quite an intuitive way to use it :D and again, it is coming with the "returning Models from calls" changes. But I'm glad you figured it out!

**Comment by felix-hilden** [comment]
I think it's worth mentioning that the changes were merged in #37, enjoy!

**Comment by marioortizmanero** [comment]
Great job!!
